Jenkinsfile

```diff
@@ -0,0 +1,84 @@
+pipeline {
+    agent any
+
+    environment {
+        GITHUB_REPOSITORY_URL   = 'https://github.com/reloadedd/UniversalStorageTool' 
+        GITHUB_ACCESS_TOKEN     = credentials('a41ba52b-a90d-40ff-9336-8d3e3ccaad50')
```

Comment:
Should this be available for the public?

Reply:
Yep. You know the url, so that's nothing secret to it. Regarding the credentials part, you know that there are some credentials (you don't know what's there, you just know the id that jenkins use to store those credentials). If you don't have access to the Jenkins server, that string `a41ba52b-a90d-40ff-9336-8d3e3ccaad50` means nothing to you. Even if you do have access, you still can't view what's inside. It's just an identifier. I could also write `credentials('Jenkins-credentials')` and would have the same effect.